test/augmentation/test_container.py

```diff
@@ -418,19 +430,19 @@ def test_inverse_and_forward_return_transform(self, random_apply, device, dtype)
             ),
             K.ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.0, return_transform=True),
             K.RandomAffine(360, p=1.0, return_transform=True),
-            data_keys=["input", "mask", "bbox", "keypoints"],
+            data_keys=["input", "mask", "bbox", "keypoints", "bbox", "BBOX_XYWH", "BBOX_XYWH"],
```

Comment:
do we have any test where the inputs are in different order ?

Reply:
The logic is to find the first "input" tag, then generate all the parameters. Then apply one by one accordingly.